documentation/sphinx/source/release-notes.rst

```diff
@@ -67,6 +70,7 @@ Other Changes
 * Added two knobs ``LOAD_BALANCE_ZONE_ID_LOCALITY_ENABLED`` and ``LOAD_BALANCE_DC_ID_LOCALITY_ENABLED`` allowing locality-based decision-making to be toggled on/off during load balancing. `(PR #1820) <https://github.com/apple/foundationdb/pull/1820>`_.
 * Ratekeeper will aggressively throttle when unable to fetch the list of storage servers for a considerable period of time. `(PR #1858) <https://github.com/apple/foundationdb/pull/1858>`_.
 * ``fdbserver`` now accepts a comma separated list of public and listen addresses. `(PR #1721) <https://github.com/apple/foundationdb/pull/1721>`_.
+* ``CAUSAL_READ_RISKY`` has been enhanced to further reduce the chance of causally inconsistent reads. `(PR #1841) <https://github.com/apple/foundationdb/pull/1841>`_.
```

Comment:
I think we decided this should note that `CAUSAL_READ_RISKY` could get more expensive than it used to be if the latency between logs and proxies was high enough.